api/spec/controllers/mno_enterprise/jpi/v1/app_instances_controller_spec.rb

```diff
@@ -47,6 +51,15 @@ module MnoEnterprise
         end
       end
 
+      context 'with unscoped data' do
+        before { allow_any_instance_of(MnoEnterprise::AppInstance).to receive(:without_tenant).and_return(true) }
+
+        it 'retrieved the app instance with the app' do
```

Comment:
I don't think this spec is testing much since you're stubbing `AppInstance#app`

It should be something like this (comments inlined):
```ruby
# remove the MnoEnterprise::AppInstance#without_tenant and MnoEnterprise::AppInstance#app stubs
context 'with unscoped data' do
  subject { get :index, organization_id: organization.id, timestamp: timestamp, unscoped: true }

  it 'retrieves the unscoped app' do
    # Test that we do the correct request to mnohub
    expect(MnoEnterprise::App).to receive(:find).with(app.id, {unscoped: true).and_return(app)
    subject
    # This test is pretty useless as we're testing something stubbed but let's keep it
    expect(app_instance.app).to eq(app)
  end
end
```